### ISSUE TITLE
Ubuntu 24.04 - crash on startup

### ISSUE THREAD

**Issue by benzkji** [nulloy/nulloy#232]
--- ~/Apps » ./Nulloy-0.9.8.7-x86_64.AppImage 
found container "GStreamer" :
* PlaybackEngine
* WaveformBuilder
found container "TagLib" :
* TagReader
* CoverReader
registering plugin: "NPlaybackEngineGStreamer"
registering plugin: "NWaveformBuilderGstreamer"
registering plugin: "NTagReaderTaglib"
registering plugin: "NCoverReaderTaglib"
detected window manager: "GNOME Shell"
realloc(): invalid pointer
Aborted (core dumped)

Any hints, or tips how to get more verbose infos welcome...

**Comment by benzkji** [comment]
It doesnt work either on Ubuntu 20.04. 
Strange. Maybe I'm just doing something wrong.

Download, set permission to execute, double click / launch in terminal.

**Comment by benzkji** [comment]
20.04:

 » ./Nulloy-0.9.8.7-x86_64.AppImage --log                                                 130 ↵
found container "GStreamer" :
* PlaybackEngine
* WaveformBuilder
found container "TagLib" :
* TagReader
* CoverReader
registering plugin: "NPlaybackEngineGStreamer"
registering plugin: "NWaveformBuilderGstreamer"
registering plugin: "NTagReaderTaglib"
registering plugin: "NCoverReaderTaglib"
detected window manager: "GNOME Shell"
^C
--- ~/Apps »

**Comment by benzkji** [comment]
@noscript any ideas?

**Comment by noscript** [comment]
Sorry for late reply. Please try https://github.com/nulloy/nulloy/releases/tag/0.9.8.14-pre

**Comment by benzkji** [comment]
No problem, thanks!  
Works no 20.04, will try this evening on 24.04...

**Comment by benzkji** [comment]
Maybe a snap would ease installation (and upgrades ;)? 
It would probably not be that hard, but it still needs to be done, tested, etc.
https://ubuntu.com/tutorials/create-your-first-snap#1-overview

It might help to get a broader audience, in the linux universe.

**Comment by benzkji** [comment]
I would be willing to help here, if you might accept a PR?

**Comment by noscript** [comment]
> Maybe a snap would ease installation (and upgrades ;)?

Snap is Ubuntu only. I'm trying to do the opposite, generalize and decentralize the distribution of Nulloy.

**Comment by vitaly-zdanevich** [comment]
> Snap is Ubuntu only

No, other distributions can install snap, for example Gentoo Linux.

**Comment by benzkji** [comment]
Kind of Ubuntu only, that's true. But for sure more generalized than .deb ;) Snap, flatpak and AppImage, in a perfect world :)

Up to you @noscript . As I read, DesktopIntegration (Open with..., open with app launcher) and such stuff seems possible with AppImage shomehow. That, and painless upgrades, with the second less important, are missing (for me).

**Comment by benzkji** [comment]
can confirm that it works on 24.04 as well.

**Comment by noscript** [comment]
Thank you for the suggestions. Yes, I plan to make AppImage updatable with file associations working but migrating to Qml and Qt6 is now a top priority. I may also reconsider snap in the future but do not hold your breath.
